liquibase-service/src/main/resources/application.yaml

```diff
@@ -0,0 +1,17 @@
+spring:
+  application:
+    name: liquibase-service
+  datasource:
+    password: ${DB_PASSWORD:-fake.fake.fake.1234}
+    username: ${DB_USERNAME:-sa}
+    url: ${DB_URL:localhost}
+    odse: ${DB_ODSE:NBS_ODSE}
+    rdb: ${DB_RDB:RDB}
```

Comment:
Small note: Does this need to point to RDB_Modern here for internal testing or are we good to keep this as RDB_Modern?

Reply:
Yeah we can control it locally by overriding this parameter. In prod deployment, we can override this in the ArgoCd